src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Godot;
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+using Newtonsoft.Json;
+using Directory = Godot.Directory;
+using File = Godot.File;
+using Path = System.IO.Path;
+
+/// <summary>
+///    A species saved by the user. Contains helper methods for saving/loading species on the disk.
+/// </summary>
+public class FossilisedSpecies
+{
+    public const string SAVE_SAVE_JSON = "save.json";
+    public const string SAVE_INFO_JSON = "info.json";
+
+    /// <summary>
+    ///   Name of this saved species on disk.
+    /// </summary>
+    public string Name { get; set; } = "invalid";
+
+    /// <summary>
+    ///   General information about this saved species.
+    /// </summary>
+    public FossilisedSpeciesInformation Info { get; set; } = new();
+
+    /// <summary>
+    ///   The species to be saved/loaded.
+    /// </summary>
+    public Species Species { get; set; } = null!;
```

Comment:
I really think you should add a constructor to force people to set this right as in my opinion this use of nullability overwrite is an unnecessary way to get around it.